packages/@aws-cdk/region-info/lib/static.generated.ts

```diff
@@ -0,0 +1,371 @@
+      "states": "states.sa-east-1.amazonaws.com"
+    },
+    "cdkMetadataResourcePresent": true
+  },
+  "us-east-1": {
+    "name": "us-east-1",
+    "partition": "aws",
+    "s3WebsiteEndpoint": "s3-website-us-east-1.amazonaws.com",
+    "servicePrincipals": {
+      "application-autoscaling": "application-autoscaling.amazonaws.com",
+      "autoscaling": "autoscaling.amazonaws.com",
+      "codedeploy": "codedeploy.us-east-1.amazonaws.com",
+      "ec2": "ec2.amazonaws.com",
+      "events": "events.amazonaws.com",
+      "lambda": "lambda.amazonaws.com",
+      "logs": "logs.us-east-1.amazonaws.com",
+      "s3": "s3.amazonaws.com",
+      "sns": "sns.amazonaws.com",
+      "sqs": "sqs.amazonaws.com",
+      "states": "states.us-east-1.amazonaws.com"
+    },
+    "cdkMetadataResourcePresent": true
+  },
+  "us-east-2": {
+    "name": "us-east-2",
+    "partition": "aws",
+    "s3WebsiteEndpoint": "s3-website.us-east-2.amazonaws.com",
+    "servicePrincipals": {
+      "application-autoscaling": "application-autoscaling.amazonaws.com",
+      "autoscaling": "autoscaling.amazonaws.com",
+      "codedeploy": "codedeploy.us-east-2.amazonaws.com",
+      "ec2": "ec2.amazonaws.com",
+      "events": "events.amazonaws.com",
+      "lambda": "lambda.amazonaws.com",
+      "logs": "logs.us-east-2.amazonaws.com",
+      "s3": "s3.amazonaws.com",
+      "sns": "sns.amazonaws.com",
+      "sqs": "sqs.amazonaws.com",
+      "states": "states.us-east-2.amazonaws.com"
+    },
+    "cdkMetadataResourcePresent": true
+  },
+  "us-west-1": {
+    "name": "us-west-1",
+    "partition": "aws",
+    "s3WebsiteEndpoint": "s3-website-us-west-1.amazonaws.com",
+    "servicePrincipals": {
+      "application-autoscaling": "application-autoscaling.amazonaws.com",
+      "autoscaling": "autoscaling.amazonaws.com",
+      "codedeploy": "codedeploy.us-west-1.amazonaws.com",
+      "ec2": "ec2.amazonaws.com",
+      "events": "events.amazonaws.com",
+      "lambda": "lambda.amazonaws.com",
+      "logs": "logs.us-west-1.amazonaws.com",
+      "s3": "s3.amazonaws.com",
+      "sns": "sns.amazonaws.com",
+      "sqs": "sqs.amazonaws.com",
+      "states": "states.us-west-1.amazonaws.com"
+    },
+    "cdkMetadataResourcePresent": true
+  },
+  "us-west-2": {
+    "name": "us-west-2",
+    "partition": "aws",
+    "s3WebsiteEndpoint": "s3-website-us-west-2.amazonaws.com",
+    "servicePrincipals": {
+      "application-autoscaling": "application-autoscaling.amazonaws.com",
+      "autoscaling": "autoscaling.amazonaws.com",
+      "codedeploy": "codedeploy.us-west-2.amazonaws.com",
+      "ec2": "ec2.amazonaws.com",
+      "events": "events.amazonaws.com",
+      "lambda": "lambda.amazonaws.com",
+      "logs": "logs.us-west-2.amazonaws.com",
+      "s3": "s3.amazonaws.com",
+      "sns": "sns.amazonaws.com",
+      "sqs": "sqs.amazonaws.com",
+      "states": "states.us-west-2.amazonaws.com"
+    },
+    "cdkMetadataResourcePresent": true
+  },
+};
+
+export = staticData;
```

Comment:
maybe just use jest snapshot test instead of checking this in (same effect). as a principle I don't like to check in generated files. It's a bad habit.

Reply:
Except this doesn't use `jest` and I don't want to code up my own snapshot tool ... again.